[2025-01-23T14:10:12.521+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: full_pipline.task45.task234_training_models.task_model2 manual__2025-01-23T14:09:42.910845+00:00 [queued]>
[2025-01-23T14:10:12.576+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: full_pipline.task45.task234_training_models.task_model2 manual__2025-01-23T14:09:42.910845+00:00 [queued]>
[2025-01-23T14:10:12.577+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-01-23T14:10:12.673+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): task45.task234_training_models.task_model2> on 2025-01-23 14:09:42.910845+00:00
[2025-01-23T14:10:12.711+0000] {standard_task_runner.py:60} INFO - Started process 9447 to run task
[2025-01-23T14:10:12.726+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'full_pipline', 'task45.task234_training_models.task_model2', 'manual__2025-01-23T14:09:42.910845+00:00', '--job-id', '178', '--raw', '--subdir', 'DAGS_FOLDER/allTasks_taskgroup.py', '--cfg-path', '/tmp/tmpxl60hhyo']
[2025-01-23T14:10:12.728+0000] {standard_task_runner.py:88} INFO - Job 178: Subtask task45.task234_training_models.task_model2
[2025-01-23T14:10:12.782+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:194: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2025-01-23T14:10:12.915+0000] {task_command.py:423} INFO - Running <TaskInstance: full_pipline.task45.task234_training_models.task_model2 manual__2025-01-23T14:09:42.910845+00:00 [running]> on host a8cdec4ae828
[2025-01-23T14:10:13.379+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='full_pipline' AIRFLOW_CTX_TASK_ID='task45.task234_training_models.task_model2' AIRFLOW_CTX_EXECUTION_DATE='2025-01-23T14:09:42.910845+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-23T14:09:42.910845+00:00'
[2025-01-23T14:10:13.608+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-01-23T14:10:13.628+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=full_pipline, task_id=task45.task234_training_models.task_model2, execution_date=20250123T140942, start_date=20250123T141012, end_date=20250123T141013
[2025-01-23T14:10:13.707+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-01-23T14:10:13.824+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
